news/35.bugfix.5

```diff
@@ -0,0 +1,3 @@
+Renamed `blacklisted_status` key to `disallowlisted_status` to be sensitive.
```

Comment:
Sorry for not saying this before you implemented it, but I think a better name would be "blocked_status". "disallow" sounds like it is related to permission, but really this is about blocking inheritance, IIRC